qa/integration-tests/src/test/java/io/camunda/it/client/SearchUserTaskTest.java

```diff
@@ -105,42 +104,26 @@ public void shouldRetrieveTaskByCandidateUser() {
 
   @Test
   public void shouldRetrieveTaskByCompletionDate() {
-    final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
-    dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+    final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+    final ZoneId utcZoneId = ZoneId.of("UTC");
 
-    final Date now = new Date();
+    final LocalDateTime now = LocalDateTime.now(utcZoneId);
```

Comment:
🔧 I think you can even do the following to obtain the current date for a specific time zone:
```java
final OffsetDateTime now = OffsetDateTime.now(ZoneId.of("UTC"))
```